### PR TITLE
Add colombia_ani_records spider

### LINKED ISSUE #823: ANI - Colombia Bulk Download Spider
**Links to bulk download and/or API endpoints**
https://apicost.azurewebsites.net/cost/records

**Link to CRM issue**
CRM-7263

**Priority**
as soon as possible

### PR DESCRIPTION
**Pull request by aguilerapy**
closes #823

### CODE REVIEW

File: kingfisher_scrapy/spiders/colombia_ani_records.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    API documentation
+      https://apicost.azurewebsites.net/cost/records
+    """
+    name = 'colombia_ani_records'
+
+    # SimpleSpider
+    data_type = 'record_package'
+
+    def start_requests(self):
+        url = 'https://apicost.azurewebsites.net/cost/records'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/colombia_ani_records.py:19]:
```suggestion
        # Extracted from https://aniscopio.ani.gov.co/datos-abiertos, 'OCDS' tab
        url = 'https://apicost.azurewebsites.net/cost/records'
```

**Review by yolile — CHANGES_REQUESTED** [review]
And you need to run `updatedocs`

File: docs/spiders.rst [hunk 2]
```diff
@@ -806,7 +813,7 @@ Mexico
 
    scrapy crawl mexico_nuevo_leon_releases
 
-.. autoclass:: kingfisher_scrapy.spiders.mexico_oaxaca_iaip.MexicoOaxacaIAIP
+.. autoclass:: kingfisher_scrapy.spiders.mexico_oaxaca_iaip.MexicoOaxacaIAIPO
```
Comment (yolile) [inline docs/spiders.rst:816]: you need to merge main into your branch

Comment (aguilerapy) [reply docs/spiders.rst]: done

**Pull request merged**